doc/howtos/building-images.rst

```diff
@@ -229,3 +229,16 @@ build configuration.
 Alternatively, ``CORE_IMAGE_EXTRA_INSTALL`` can also be used. The
 difference is that this will also affect the initramfs images, which is
 often not intended.
+
+Automatic Cleaning of Old Image Before New One Get Built
+--------------------------------------------------------
+
+Every image built gets copied in the deploy directory. Further, a new build
```

Comment:
Every image build gets copied into the deploy directory. As you're developing, these repeated builds will start accumulating and use up more and more disk space. You can save disk space by removing previous images before the new one is built by adding this line to your local.conf:  RM_OLD_IMAGE = "1"